.github/workflows/ci-windows.yml

```diff
@@ -24,3 +24,23 @@ jobs:
         mingw32-make
         mingw32-make install
         cd -
+    - name: Build installer
+      run: |
+        pip install mingw-ldd
+        MINGW_SYSROOT_BIN="/usr/i686-w64-mingw32/sys-root/mingw/bin" concordance/win/make_installer.py
+    - uses: actions/upload-artifact@v2
+      with:
+        name: windows-installer
+        path: concordance/win/concordance-installer.exe
+  test:
+    needs: build
+    runs-on: windows-latest
+    steps:
+    - uses: actions/download-artifact@v2
+      id: download
+      with:
+        name: windows-installer
+    - name: Install and test
+      run: |
+        ${{steps.download.outputs.download-path}}\concordance-installer.exe /S
+        "C:\Program Files (x86)\Concordance\concordance.exe --version"
```

Comment:
<3

Reply:
I'm not convinced that this actually worked though as I don't see anything in the output.  Guess I should have marked this [WIP] as I was still working on it.  :-)

Reply:
You know you can mark a PR as "draft" in GH as well, right?

Reply:
![image](https://user-images.githubusercontent.com/938467/150701552-02b965d1-76f2-45e1-aa79-c49a5913984a.png)

Reply:
Yeah, I knew that but I couldn't figure out how to do it.  Figured it out now.